bin/commands/init/index.sh

```diff
@@ -93,7 +93,9 @@ if [ "${ZWE_CLI_PARAMETER_SKIP_SECURITY_SETUP}" != "true" ]; then
   zwecli_inline_execute_command init apfauth
   zwecli_inline_execute_command init security
 fi
-zwecli_inline_execute_command init certificate
+if [ "${ZWE_CLI_PARAMETER_CREATE_CERTIFICATE}" != "true" ]; then
```

Comment:
1. Should check for "is equal": `if [ "${ZWE_CLI_PARAMETER_CREATE_CERTIFICATE}" = "true" ]; then`
2. This is a braking change as the new default is "Do not process certificates".